README.md

```diff
@@ -598,6 +598,23 @@ Other Style Guides
   var subtract = Function('a', 'b', 'return a - b');
   ```
 
+- [7.11](#7.11) <a name='7.11'></a> Avoid defaulting variables using `||`.
+
+  > Why? Because `||` does not check for strict equality (i.e. `===` vs. `==`). If a passed argument is falsy (`0`, `false`, `null`, `''`...), it will be defaulted while it shouldn't be.
+
+  ```javascript
+  // bad
+  function f(arg, optionalArg, opts = {}) {
+      const message = optionalArg : 'not set';  // what if arg === ''?
+      const rate = opts.rate || 0.1;            // what if arg === 0.0?
+  }
+
+  // good
+  function f(arg, optionalArg = 'not set', opts = {}) {
+      const rate = opts.rate !== undefined ? opts.rate : 0.1;
```

Comment:
Oh, I see!

This is, in my opinion, an orthogonal problem. Checking input validity is up to the project error-handling guidelines (checking before calling *vs.* checking the arguments for example). But preserving input data (i.e. do not transform `0` into `'default'`) should be ensured in all cases.